components/sensor/absolute_humidity.rst

```diff
@@ -0,0 +1,60 @@
+Absolute Humidity
+=================
+
+.. seo::
+    :description: Instructions for setting up absolute humidity calculation
+    :image: water-drop.svg
+
+The ``absolute_humidity`` platform allows you to calculate absolute humidity from air temperature and relative humidity.
+
+See the links at the bottom of the page for details on absolute humidity and the different saturated vapor pressure equations.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: absolute_humidity
+        name: Absolute Humidity
+        temperature: air_temperature
+        humidity: relative_humidity
+
+      # Use any temperature and relative humidity source, e.g. a BME280:
+      - platform: bme280
+        temperature:
+          name: Temperature
+          id: air_temperature
+        pressure:
+          name: Air Pressure
+        humidity:
+          name: Relative Humidity
+          id: relative_humidity
```

Comment:
Lets just leave this as a blank template as people will copy paste this and it wont match their setup
```suggestion
      - platform: ...
        temperature:
          name: Temperature
          id: air_temperature
        humidity:
          name: Relative Humidity
          id: relative_humidity
```